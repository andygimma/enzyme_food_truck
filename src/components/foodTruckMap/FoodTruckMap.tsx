import { MapContainer, TileLayer, Popup, Marker } from "react-leaflet";
import styled from "styled-components";

const Container = styled.div`
  width: 100%;
`;

const FoodTruckMap = () => {
  return (
    <Container>
      <MapContainer
        center={[40.735863, -73.991083]}
        zoom={13}
        scrollWheelZoom={false}
        style={{ height: "80vh", borderRadius: 12 }}
      >
        <TileLayer
          attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
          url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
        />

        <Marker position={[40.735863, -73.991083]}>
          <Popup>
            A pretty CSS3 popup. <br /> Easily customizable.
          </Popup>
        </Marker>
      </MapContainer>
    </Container>
  );
};

export default FoodTruckMap;
