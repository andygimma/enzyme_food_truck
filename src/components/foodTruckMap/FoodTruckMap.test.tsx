import renderer from 'react-test-renderer';
import FoodTruckMap from './FoodTruckMap';

// NOTE I use snapshots sparingly. Generally only for a component that is unlikely to be changed on purpose.
// An example could be our map container
it('renders correctly', () => {
    const tree = renderer
        .create(<FoodTruckMap />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});