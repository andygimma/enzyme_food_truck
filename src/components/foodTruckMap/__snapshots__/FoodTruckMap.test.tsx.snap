// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bczRLJ ekBFnZ"
>
  <div
    style={
      Object {
        "borderRadius": 12,
        "height": "80vh",
      }
    }
  />
</div>
`;
